.github/workflows/reusable_publish_packages.yaml

```diff
@@ -0,0 +1,111 @@
+# This is a reusable workflow used by master and release CI
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: 'Falco version extracted from userspace/falco/config_falco.h'
+        required: true
+        type: string
+      bucket:
+        description: bucket suffix for packages
+        required: false
+        default: ''
+        type: string
+        
+env:
+  AWS_S3_REGION: eu-west-1
+
+jobs:
+  publish-packages:
+    runs-on: ubuntu-latest
+    container: docker.io/centos:7
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+    
+      - name: Install dependencies
+        run: |
+          yum install epel-release -y
+          yum update -y
+          yum install rpm-sign expect which createrepo gpg python python-pip -y
+          pip install awscli==1.19.47
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: "arn:aws:iam::292999226676:role/github_actions-falco${{ inputs.bucket }}-s3"
```

Comment:
The role is either `github_actions-falco-s3` or `github_actions-falco-dev-s3`; see https://github.com/falcosecurity/test-infra/pull/1102.

The dev one can only push dev packages.